source/style-guide/style/metadata.txt

```diff
@@ -0,0 +1,19 @@
+.. _metadata:
+
+=============
+Metadata
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Use ``.. meta::`` to add meta tags to a page. Adding meta tags aids in 
+searches both internally and via the Internet.
+
+.. code-block:: rst
+     
+   .. meta::
+      :description: read concern, local read concern, read isolation, 
+      transactions, multi-document transactions 
+
```

Comment:
Possible future work:

It might be worth reaching out to one of our SEO experts for guidance on effective meta tags. For example, should we avoid repeating words across multiple tags? Are there any helpful guidelines to be aware of?

Reply:
I think we have a document for this already, just need to find and link it.

Reply:
Sure, I think it makes sense to expand the content, but as Jeff mentioned, separate from this review. I think that document looks great. I'm appending some info at the end from a related discussion I had with Docs Platform a few months ago.